package.json

```diff
@@ -39,8 +39,21 @@
     "rxjs": "^7.8.0",
     "uuid": "^9.0.1"
   },
+  "resolutions": {
+    "**/cross-spawn": "^7.0.5",
+    "**/fork-ts-checker-webpack-plugin": "^6.5.3",
+    "**/ip": "https://registry.npmjs.org/neoip/-/neoip-2.1.0.tgz",
+    "**/micromatch": "^4.0.8",
+    "**/nanoid": "^3.3.8",
+    "**/path-to-regexp": "^0.1.12",
+    "**/postcss": "^8.4.49",
+    "**/resolve-url-loader": "^5.0.0",
+    "**/semver": "^7.6.3",
+    "**/@svgr/webpack": "^8.1.0",
+    "**/webpack": "^5.97.1"
```

Comment:
uff... this seems to be getting out of hand quickly.

Is there a reason to use "^" which locks the major version, instead of ">=" which says "higher than this vulnerable version"?

Reply:
My reasoning is that major versions can introduce breaking changes that affect the build, tools and configs.